guides/process/how_to_timesheet.md

```diff
@@ -0,0 +1,85 @@
+# How to timesheet
+
+## General
+
+Made Tech hours are 35 per week, 7 hours per day full time. Part-time hours are prorated and reflected in Harvest according to your agreed hours. 
+
+Only billable team members need to complete a timesheet and you will be notified at your onboarding session if you need to complete one. The timesheet must be completed to account for all of your time in the working week. 
+
+Time can be completed in advance but in your unplanned absence your delivery manager can complete your timesheet. 
+
+## When to submit your time
+
+Time sheets need to be submitted weekly every Friday by Midday. It needs to be completed weekly because it is likely to be a truer reflection of time and we are able to accurately invoice the client using this information.
```

Comment:
Not sure midday is normally capitalised...
```suggestion
Time sheets need to be submitted weekly every Friday by midday. It needs to be completed weekly because it is likely to be a truer reflection of time and we are able to accurately invoice the client using this information.
```